packages/components/entity-list/src/EntityListItem/EntityListItem.tsx

```diff
@@ -219,17 +221,33 @@ export const EntityListItem = ({
                 </Text>
               )}
             </Flex>
-            {description && (
-              <Text
-                as="p"
-                lineHeight="lineHeightM"
-                fontColor="gray900"
-                isTruncated
-                className={styles.description}
-              >
-                {description}
+            <Flex marginTop="spacing2Xs">
+              {entityType && (
+                <Text
+                  as="span"
+                  lineHeight="lineHeightM"
+                  fontColor="gray600"
+                  className={styles.entityType}
+                  isTruncated
```

Comment:
```suggestion
```

Reply:
I'm thinking the entity type should always be fully shown, the alternative might look weird.

Reply:
Copy pasta mistake 😂